Gemfile

```diff
@@ -12,7 +12,7 @@ gem 'dotenv-rails', '~> 2.4'
 gem 'enum_help'
 gem 'figaro'
 gem 'httparty'
-gem 'identity-hostdata', github: '18F/identity-hostdata', branch: 'master'
+gem 'identity-hostdata', git: 'https://github.com/18F/identity-hostdata.git', branch: 'master'
```

Comment:
I did this since `github:` results in a url with the scheme `git:` which CodeClimate complains is insecure.